setup.cfg

```diff
@@ -0,0 +1,38 @@
+[tool:pytest]
+addopts = --strict-markers
+norecursedirs = .* build dist CVS _darcs {arch} *.egg venv env virtualenv
+
+
+[mypy]
+files = commitizen, tests
+ignore_missing_imports = true
+# disallow_untyped_calls = True
+# disallow_untyped_defs = True
```

Comment:
I'd be fine with `ignore_missing_imports` and `disallow_untyped_defs`, what'd be the benefit for the others?

Reply:
The basic idea of all others is the same as `disallow_untyped_defs` which is to enforce a stricter type annotation.

Actually, they are what I just learned from watching the excellent talk by Vita Smid in EuroPython 2019 haha
[Static typing: beyond the basics of def foo(x: int) -str:](https://lee-w.github.io/pycon-note/posts/europython-2019/2020/03/static-typing-beyond-the-basics-of-def-foo-x-int-str/)